<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="lbl_4">
      <property name="text">
       <string>Описание вкуса</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_3">
      <property name="text">
       <string>Молотый/в зернах</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_2">
      <property name="text">
       <string>Степень обжарки</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_1">
      <property name="text">
       <string>Название сорта</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_6">
      <property name="text">
       <string>Объем упаковки</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_5">
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_sort"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_degree"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="le_ground"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="le_taste"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="le_cost"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="le_volume"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
